community/examples/hpc-slurm-local-ssd-v5-legacy.yaml

```diff
@@ -37,6 +37,14 @@ deployment_groups:
     settings:
       local_mount: /home
 
+  - id: startup
+    source: ./modules/scripts/startup-script
+    settings:
+      local_ssd_filesystem:
+        fs_type: "ext4"
+        mountpoint: "/mnt/localssd"
+        permissions: "0755"
+
```

Comment:
Can you please add a comment here identical to #2871 about the loss of data when shutting nodes down without specifying local SSD data migration?

Reply:
Added!